test/modules/post/test/services.rb

```diff
@@ -66,6 +66,9 @@ def test_start
 
   def test_list
     return skip('session platform is not windows') unless session.platform == 'windows'
+    if !session.commands.include?(Rex::Post::Meterpreter::Extensions::Stdapi::COMMAND_ID_STDAPI_RAILGUN_API)
+      return skip('reg query support skipped for now as the query takes more than two minutes')
```

Comment:
Skipping this test for now if railgun isn't present; @bwatters-r7 and I both spotted that the list services command is extremely slow via reg query/cmd_exec